qa/tasks/cephfs/kernel_mount.py

```diff
@@ -61,12 +63,17 @@ def mount(self, mount_path=None, mount_fs_name=None):
         if mount_path is None:
             mount_path = "/"
 
-        opts = 'name={id},secretfile={secret},norequire_active_mds'.format(id=self.client_id,
-                                                      secret=secret)
+        opts = 'name={id},secretfile={secret},norequire_active_mds'.format(
+                id=self.client_id,
+                secret=secret)
+
+        if self.no_rsize:
+            opts += ",rsize=0"
```

Comment:
rsize for kernel client is readahead size.  the option for disabling 'rstats for dir size' is norbytes